app/main.py

```diff
@@ -1 +1,25 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    parts = command.split()
+    if len(parts) != 3 or parts[0] != "mv":
+        raise ValueError("Invalid command format."
+                         " Expected format: 'mv source destination'")
+
+    source_path = parts[1]
+    destination_path = parts[2]
```

Comment:
Consider using `os.path.join` to construct paths. This ensures cross-platform compatibility as per the checklist requirement.